ledger-api/grpc-definitions/com/daml/ledger/api/v1/update_service.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/ledger_offset.proto";
+import "com/daml/ledger/api/v1/transaction.proto";
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+import "com/daml/ledger/api/v1/transfer.proto";
+
+option java_outer_classname = "UpdateServiceOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Allows clients to read updates (transactions and transfers) from the ledger.
+service UpdateService {
+
+  // Read the ledger's filtered transaction stream and related transfers for a set of parties.
```

Comment:
Alternatively, if empty transactions are still emitted that would also work.

Reply:
Yes, I had a line in my notes that you will come back to me on the initial idea that we add the transfers simply to the flat stream, so here we go :) I will reach out to discuss this in detail.
Initial thoughts:
* as long as this is a new endpoint, so everything is possible
* still it would be wise to be as close to capabilities of 2.0 as possible
* I would really welcome not going in a direction where we have to define Tree stream with interface subscription, because that is conceptually different to what we have already

Reply:
As discussed offline, the preferred approach would be:
* supporting interface projections on transaction trees
* and/or supporting interface projections on point wise tx lookups